rustls/src/crypto/hmac.rs

```diff
@@ -20,23 +20,30 @@ pub trait Hmac: Send + Sync {
 }
 
 /// A HMAC tag, stored as a value.
+///
```

Comment:
Thinking of my latter feedback for the non-secret headline docstring, perhaps this one should be:

> A secret HMAC tag, stored as a value.